dockerfiles/theia/src/entrypoint.sh

```diff
@@ -76,7 +76,8 @@ if [ "${NOCDN}" == "true" ]; then
 fi
 shopt -u nocasematch
 
-# run che
+# run Che Theia
+export THEIA_WEBVIEW_EXTERNAL_ENDPOINT=$(node get-webview-route.js)
```

Comment:
question: we don't have the endpoint as environment variable ?

Reply:
I mean, che-server is not providing that information ?

Reply:
No, we don't have external route in env vars.